docs/design/metadata-storage.md

```diff
@@ -44,7 +44,9 @@ See [Metadata storage configuration](../configuration/index.md#metadata-storage)
 
 ## Available metadata stores
 
-Druid supports Derby, MySQL, and PostgreSQL for storing metadata. 
+Druid supports Derby, MySQL, and PostgreSQL for storing metadata.
+
+To avoid issues with upgrades that require scehma changes to a large metadata table, chose a metadata store version that supports instant ADD COLUMN semantics.
```

Comment:
Can you also add to see the database-specific docs on any specific guidance about versions? Also, we should say "prefer' instead of 'chose'.

Reply:
It is unusual usage to instruct someone to "prefer" something. I changed to "consider".